visualization/tools/pcd_viewer.cpp

```diff
@@ -422,6 +428,19 @@ main (int argc, char** argv)
     if (pcd.read (argv[p_file_indices.at (i)], *cloud, origin, orientation, version) < 0)
       return (-1);
 
+    // Calculate transform if available.
+    if (pose_x.size () > i && pose_y.size () > i && pose_z.size () > i &&
+        pose_roll.size () > i && pose_pitch.size () > i && pose_yaw.size () > i)
+    {
```

Comment:
sorry I just saw this nice addition.
wouldn't it make sense if you could only give position parameters?
I think this is a very common application, when you for example want to show multiple object pcds in one run, just displaced by lets say 1 meter from each other?
For that a very nice addition would be to check if the pose_x,y,z sizes are one, i.e. only one displacement vector **d**
In that case the standard behaviour would be to displace each pointcloud with index *Ind* by **d** * *Ind* ?
What do you think?